website/client/js/services/costumeServices.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+(function(){
+  angular
+    .module('habitrpg')
+    .factory('Costume', costumeFactory);
+
+  costumeFactory.$inject = [
+    'Content'
+  ];
+
+  function costumeFactory(Content) {
+    
+    function formatAnimal(name, type) {
+      if(type === 'pet' && name in Content.petsInfo) {
+        return Content.petsInfo[name].text();
+      } else if(type === 'mount' && name in Content.mountsInfo) {
+        return Content.mountsInfo[name].text();
+      }
+    }
+
+    function bgString(set, bg) {
+      var setString = window.env.t(set);
```

Comment:
I mean one that is flat
On Wed, Sep 14, 2016 at 9:08 AM Blade Barringer <notifications@github.com>
wrote:

> In website/client/js/services/costumeServices.js
> <https://github.com/HabitRPG/habitrpg/pull/7768#discussion_r78755239>:
>
> > +  costumeFactory.$inject = [
> > +    'Content'
> > +  ];
> > +
> > +  function costumeFactory(Content) {
> > +
> > +    function formatAnimal(name, type) {
> > +      if(type === 'pet' && name in Content.petsInfo) {
> > +        return Content.petsInfo[name].text();
> > +      } else if(type === 'mount' && name in Content.mountsInfo) {
> > +        return Content.mountsInfo[name].text();
> > +      }
> > +    }
> > +
> > +    function bgString(set, bg) {
> > +      var setString = window.env.t(set);
>
> Correct, organized by set. I am proposing a flat gear object, similar to
> how we have the gear and flat gear objects.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/HabitRPG/habitrpg/pull/7768/files/72a6d73a9be23fd72ba98fbe74b2f14fbc840bfb#r78755239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALz_81SJHQkvEdqTVUYbrC3GU7UMiXtAks5qp__IgaJpZM4JIjd_>
> .
>